packages/svelte/package.json

```diff
@@ -2,45 +2,48 @@
   "name": "@tolgee/svelte",
   "version": "4.8.0",
   "scripts": {
-    "build": "svelte-kit package",
+    "dev": "vite dev",
+    "build": "svelte-kit sync && svelte-package",
+    "develop": "sleep 5 && nodemon --exec svelte-package",
     "clean": "rm -rf build build-e2e dist dist-e2e lib .svelte-kit",
-    "preview": "svelte-kit preview",
-    "develop": "sleep 5 && nodemon --exec svelte-kit package",
+    "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
+    "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
     "test": "jest src",
-    "check": "svelte-check --tsconfig ./tsconfig.json",
-    "check:watch": "svelte-check --tsconfig ./tsconfig.json --watch",
     "lint": "prettier --ignore-path .gitignore --check --plugin-search-dir=. . && eslint --ignore-path .gitignore .",
     "format": "prettier --ignore-path .gitignore --write --plugin-search-dir=. .",
     "prepare": "svelte-kit sync"
   },
   "dependencies": {
-    "@tolgee/core": "^4.8.0",
-    "@tolgee/ui": "^4.8.0"
+    "@tolgee/core": "workspace:^4.8.0",
```

Comment:
I haven't seen any other usage of this in the workspace. Should I remove it?

Reply:
Yes you can probably remove it. Also plz remove "workspace:". I'm afraid it's special pnpm thing, which would break our release process.

Reply:
done